parser/parser/include/ruby_parser/driver.hh

```diff
@@ -424,6 +424,10 @@ public:
     void local_push();
     void local_pop();
 
+    const std::vector<std::pair<size_t, size_t>> &get_comment_locations() const {
+        return lex.comment_locations;
```

Comment:
I wanted to convert this `std::pair` into a `LocOffsets` here but I was having trouble including it with a `'core/LocOffsets.h' file not found`. I can refactor this if I'm unblocked.

Reply:
If you add a dependency on `//core` in `parser/parser/BUILD` for the `parser` target, it should be available.

Reply:
Done. I can revert this change if the separation in compilation is more desired than using `LocOffsets`.